src/seeds.accounts.cpp

```diff
@@ -1521,7 +1530,7 @@ void accounts::evaldemote (name to, uint64_t start_val, uint64_t chunk, uint64_t
     transaction tx;
     tx.actions.emplace_back(next_execution);
     tx.delay_sec = 1;
-    tx.send(next_value + 1, _self); 
+    tx.send(next_value + 1, _self);
   }
```

Comment:
I think this also needs to call mimic, with remove?

Reply:
That's true, sorry I am going to fix it.